compiler/luci/pass/src/RemoveRedundantTranspose.cpp

```diff
@@ -100,13 +104,13 @@ namespace luci
  *  AFTER
  *      <Optional Case>
  *
- *          |                 |                   |
- *    [CircleNode]      [CircleConst]             |
- *     (main_node)     (new_const_node)           |
- *           \               /           or  [CircleNode]
- *           [CircleTranspose]                (main_node)
- *              (pred_node)                       |
- *                   |                            |
+ *                 |                 |                    |
+ *           [CircleNode]      [CircleConst]              |
+ *            (main_node)     (new_const_node)            |
+ *                 / \               /           or  [CircleNode]
+ * [CircleTranspose] [CircleTranspose]                (main_node)
+ *    (pred_node)    (new_trans_node)                     |
+ *                          |                             |
```

Comment:
minor thing, `|` after `(pred_node)` is removed. is `(pred_node)` disconnected? or if not then let's add `|` after `(pred_node)`

Reply:
If `pred_node` have successor more than one(not `target_node` only), then it could connect to that node. otherwise, it will not  connected to any node. so, I couldn't draw any connection line.(just I think)